README.md

```diff
@@ -19,5 +19,31 @@ Example from `usage-editable.py`
 $ pip install dash-table-experiments
 ```
 
+### Usage with Callbacks
+Per [this Dash community answer](https://community.plot.ly/t/dash-datatable-using-callbacks/6756/2), to use callbacks with `dash-table-experiments` there are two key steps (for a full working example see [usage-callback.py](./usage-callback.py)):
+
+```
+# 1. Declare the table in app.layout
+dt.DataTable(
+    rows=[{}], # initialise the rows
+    row_selectable=True,
+    filterable=True,
+    sortable=True,
+    selected_row_indices=[],
+    id='datatable'
+)
+
+# 2. Update rows in a callback
+@app.callback(Output('datatable', 'rows'), [Input('field-dropdown', 'value')])
+def update_datatable(user_selection):
+    '''
+    For user selections, return the relevant table
+    '''
+    if user_selection == 'Summary':
+        return DATA.to_dict('records')
+    else:
+        return SOMEOTHERDATA.to_dict('records')
+```
+
```

Comment:
now that we've added context to this example, would you mind adding a little context to the next two examples that are listed below (`usage.py` and `usage-editable.py`). Something like 
> For usage with row-selection, see `usage.py`. For usage with editable cells, see `usage-editable.py`